library/Fission/Prelude.hs

```diff
@@ -6,6 +6,7 @@ module Fission.Prelude
   , module Data.Aeson
   , module Data.Has
   , module Data.Maybe
+  , module Data.WorldPeace
```

Comment:
Really badly named package, but this is the one that gives us `OpenUnion`

Reply:
lmao Haskell community never fails to impress